en/docs/ai-gateway/1.0.0/llm-proxy/quick-start-guide.md

```
@@ -63,6 +66,18 @@ docker compose up -d
 curl http://localhost:9094/health
 ```
 
+!!! tip "Port 8080, 8443, 9090, or 9094 already taken?"
+    If the start command fails with a port binding error, identify what is already listening on the default ports:
+
+    ```bash
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)